baselines/previously_seen_aws_cross_account_activity___dm.yml

```diff
@@ -0,0 +1,32 @@
+name: Previously Seen AWS Cross Account Activity - DM
+id: 0de7ce99-ab0a-41fe-9624-345df83f08cc
+version: 1
+date: '2020-05-28'
+description: This datamodel search looks for **AssumeRole** events where the requesting account
+  differs from the requested account, then writes these relationships to a lookup
+  file.
+how_to_implement: You must install the AWS App for Splunk (version 5.1.0 or later)
+  and Splunk Add-on for AWS (version 4.4.0 or later), then configure your CloudTrail
+  inputs. You must also be using CIM version x.x.x, which contains updates to the Authentication
+  data model for cloud use cases. Validate the user name entries in `previously_seen_aws_cross_account_activity.csv`,
+  a lookup file created by this support search.
+author: Rico Valdezy, Splunk
+search: '`cloudtrail` eventName=AssumeRole | spath output=requestingAccountId path=userIdentity.accountId
+  | spath output=requestedAccountId path=resources{}.accountId | search requestingAccountId=*
+  | where requestingAccountId!=requestedAccountId | stats earliest(_time) as firstTime
+  latest(_time) as lastTime by requestingAccountId, requestedAccountId | outputlookup
+  previously_seen_aws_cross_account_activity | stats count'
+
+
+search: '| tstats earliest(_time) as firstTime latest(_time) as lastTime from
```

Comment:
i would drop_dm_object everywhere instead of rename

Reply:
Let's discuss, but don't disagree